RockWeb/Blocks/Groups/GroupRegistration.ascx.cs

```diff
@@ -574,6 +575,11 @@ private bool CheckSettings()
 
             tbEmail.Required = _autoFill;
 
+            string registerButtonText = GetAttributeValue( "RegisterButtonAltText" );
+            if ( string.IsNullOrWhiteSpace( registerButtonText ) )
+                registerButtonText = "Register";
```

Comment:
@treyhendon, just a [coding-standard](https://github.com/SparkDevNetwork/Rock/wiki/Coding-standards) note, we always use braces on our if/else statements even if it's just one-line :). I'll update this one after the merge. Thanks again for jumping in and doing the pull requests.
-David